### ISSUE TITLE
 API Graphql  : Parameter count xx is greater than max allowed 60

### ISSUE THREAD

**Issue by mickadoua** [aws-amplify/amplify-js#1595]
**Describe the bug**
when I run Amplify push i received  
`Template format error: Parameter count 68 is greater than max allowed 60`
if I remove one @connection, the number decrease to 64, and it i remove another couple of @connection, its work 

**To Reproduce**
1) amplify add API and choose Graphlql API
2) edit schema.graphql 
```
type Car @model @searchable{
  id: ID!
  brand: String!
  plate: String!
  driver: Driver @connection(name: "DriverCar")
}

type Driver @model @searchable{
  id: ID!
  name: String!
  #location: Coords
  latitude: Float
  longitude: Float
  available: Boolean
  active: Boolean
  car: Car @connection(name: "DriverCar")
  courses: [Course] @connection(name: "DriverCourse")
}
type Client @model{
  id: ID!
  name: String!
  #location: Coords
  latitude: Float
  longitude: Float
  active: Boolean
  courses: [Course] @connection(name: "ClientCourse")
}

type Course @model{
  id: ID!
  date: String
  client: Client @connection(name: "ClientCourse")
  driver: Driver @connection(name: "DriverCourse")
  rating: Float
  reviewText: String
}
```
3) run amplify push 

**Expected behavior**
working :D

**Desktop (please complete the following information):**
     "aws-amplify": "^1.0.8",
    "aws-amplify-react-native": "^2.0.0",

Thank you

**Comment by kaustavghosh06** [comment]
@mickadoua Whats the Amplify CLI version that you're using? This issue was fixed with the Amplify CLI version -> 0.1.18

**Comment by mickadoua** [comment]
@kaustavghosh06  Hello, 
I dont know which version I had, but after update to 0.1.18, its working, 
thank you !
